src/tests/cucumber/features/steps/simulator_utils.py

```diff
@@ -35,7 +55,7 @@ def activate_simu_outputs(context):
 
 
 def build_antares_solver_command(context):
-    command = [SOLVER_PATH, "-i", str(context.study_path)]
+    command = [SOLVER_PATH, "-i", f'"{context.study_path}"']
```

Comment:
Please revert all other changes and keep only this one.

Reply:
I have removed the copyright notice but other changes are legitimate.